tests/Fixtures/TestBundle/Entity/DummyAliasIdType.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\Entity;
```

Comment:
do you really need a doctrine entity ? can you create a simple dto inside ApiResource instead?